README.md

```diff
@@ -351,6 +351,15 @@ stepFunctions:
   validate: true
 ```
 
+### Disable Output Cloudformation Outputs section
+
+Disables output in Outputs section.
```

Comment:
It may lack the explanation in what kind of situation would it be possible to use this.
What about updating something like this?

```
Disables output in the CloudFormation Outputs section.  if you define many state-machines in serverless.yml, there is a possibility of reaching out a CloudFormation limits in which the maximum number of outputs is 60. If you define `noOutput: true`, you can prevent automatically output by this plugin.
```